arm-asazure/2016-05-06-preview/swagger/asazure.json

```diff
@@ -0,0 +1,489 @@
+            "schema": { "$ref": "#/definitions/CancellationToken" },
+            "description": "Cancellation token"
+          },
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Subscription Id for the subscription in which the operation is executed"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Region in which the server operation was executed"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/Object" }
+          }
+        }
+      }
+    },
+    "/resourceprovider/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/locations/{location}/operationresults/{operationId}": {
+      "get": {
+        "description": "Gets result of the specified operation",
+        "tags": [ "RpProxy" ],
+        "operationId": "RpProxy_GetOperationResultAsync",
+        "consumes": [ "application/json", "text/json"],
+        "produces": [ "application/json", "text/json"],
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Identifier for the operation for which to get status"
+          },
+          {
+            "name": "cancellationToken",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/CancellationToken" },
+            "description": "Cancellation token"
+          },
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Subscription Id for the subscription in which the operation is executed"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Region in which the server operation was executed"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/Object" }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CancellationToken": {
+      "description": "Cancellation token specification",
+      "type": "object",
+      "properties": {
+        "IsCancellationRequested": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "CanBeCanceled": {
+          "type": "boolean",
```

Comment:
please make sure all the model properties have a description. The description becomes the client library documentation. There is an ongoing project which will convert the swagger spec into an MSDN REST API style doc. Thus ensuring accurate description is of paramount importance.